### DISCUSSION TITLE
FAQ accordion

### DISCUSSION THREAD

**Discussion by heidimia** [community/community#48876] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How can I display the rows collapsed?

https://github.com/pnp/List-Formatting/tree/master/view-samples/faq-accordion

For example:
<img width="184" alt="FAQ" src="https://user-images.githubusercontent.com/123971695/222099439-beb8cf8c-5656-4393-81b0-20442a4ed2ba.png">

**Comment by Anmol-Baranwal** [comment, marked as answer]
Go through this [repo](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code)

I am sure, you would find your solution here.

**Reply by heidimia** [reply]
Thank you so much! Unfortunately I am a beginner.
I used this JSON code which works fine. But I only want to show the view collapsed and not opened. I want you to be able to expand it yourself.
How do I have to set this in the JSON code?

{
  "$schema": "https://developer.microsoft.com/json-schemas/sp/v2/row-formatting.schema.json",
  "hideSelection": true,
  "hideColumnHeader": true,
  "hideListHeader": true,
  "groupProps": {
    "headerFormatter": {
      "elmType": "div",
      "attributes": {
        "class": "sp-row-card"
      },
      "style": {
        "color": "black",
        "background-color": "#FAF9F9",
        "flex-grow": "1",
        "display": "flex",
        "flex-direction": "row",
        "box-sizing": "border-box",
        "padding": "4px 8px 5px 8px",
        "border-radius": "6px",
        "align-items": "center",
        "flex-wrap": "nowrap",
        "overflow": "auto",
        "margin": "1px 4px 4px 1px"
      },
      "children": [
        {
          "elmType": "img",
          "style": {
            "max-width": "24px",
            "max-height": "24px",
            "margin-top": "2px",
            "border-radius": "2px"
          }
        },
        {
          "elmType": "div",
          "children": [
            {
              "elmType": "span",
              "style": {
                "padding": "5px 5px 5px 5px",
                "font-weight": "500",
                "font-size": "15px"
              },
              "txtContent": "@group.fieldData.displayValue"
            }
          ]
        },
        {
          "elmType": "div",
          "children": [
            {
              "elmType": "div",
              "style": {
                "display": "flex",
                "flex-direction": "column",
                "justify-content": "center"
              }
            }
          ]
        }
      ]
    }
  },
  "rowFormatter": {
    "elmType": "div",
    "attributes": {
      "class": "ms-bgColor-themeLighterAlt"
    },
    "style": {
      "justify-content": "start",
      "color": "ms-fontColor-Primary",
      "display": "flex",
      "padding": "1px 8px 5px 20px",
      "border-radius": "6px",
      "max-width": "905px",
      "margin": "1px 4px 4px 60px"
    },
    "children": [
      {
        "elmType": "div",
        "style": {
          "text-align": "left",
          "font-weight": "400",
          "font-size": "15px"
        },
        "children": [
          {
            "elmType": "span",
            "style": {
              "margin-top": "8px",
              "padding-left": "1px",
              "color": "black"
            },
            "attributes": {
              "class": "ms-fontSize-m",
              "iconName": "Lightbulb"
            }
          },
          {
            "elmType": "span",
            "attributes": {
              "class": "sp-row-listPadding"
            },
            "txtContent": "Answer",
            "style": {
              "font-weight": "500",
              "color": "black",
              "padding-left": "4px"
            }
          },
          {
            "elmType": "div",
            "attributes": {
              "class": "sp-row-listPadding"
            },
            "txtContent": "[$Answer]",
            "style": {
              "flex-grow": "1",
              "display": "flex",
              "flex-direction": "column",
              "flex-wrap": "nowrap",
              "align-items": "center",
              "max-width": "900px",
              "min-width": "205px",
              "margin-top": "8px",
              "margin-bottom": "8px"
            }
          },
          {
            "elmType": "button",
            "customRowAction": {
              "action": "defaultClick"
            },
            "txtContent": "Show more",
            "attributes": {
              "class": "sp-row-button"
            },
            "style": {
              "display": "=if([$ShowMore] == 'Yes', 'block', 'none')",
              "margin-bottom": "8px",
              "color": "white",
              "background-color": "#0077DB"
            }
          }
        ]
      }
    ]
  }
}

**Reply by Anmol-Baranwal** [reply]
You can use details and summary tags if you are working in markdown format. I will have to see about JSON.

**Reply by Anmol-Baranwal** [reply]
So, if there are no further doubts. Could you mark my answer as answered and let others know that question has been answered.

**Reply by heidimia** [reply]
Sorry, I'm a beginner so unfortunately I don't know how to adjust this in this JSON-Code so you can see the content collapsed.

This is how I would like to see it and open the tabs myself:
<img width="178" alt="image" src="https://user-images.githubusercontent.com/123971695/222653144-9db4afd2-4596-4111-95b8-e93d35758620.png">
